admin/assets/javascripts/discourse/templates/update-index.hbs

```diff
@@ -1,4 +1,21 @@
-<h1>{{i18n "admin.docker.update_title"}}</h1>
+<div class="updates-heading">
+  <h3>{{i18n "admin.docker.update_title"}}</h3>
+  {{#unless this.outdated}}
+    <button
+      disabled={{this.upgradeAllButtonDisabled}}
```

Comment:
We should definitely be changing the code variables and functions from `upgrade` to `update` as well, otherwise it is confusing

Reply:
And CSS classes

Reply:
Definitely. But I thought we should do it in a seperate PR. Since the scope of that PR is little different from this one. We will be just renaming the variable names.

Reply:
OK fair enough you can do this in a followup PR :+1: